lib/VideoPlayer/VideoPlayer.less

```diff
@@ -45,32 +45,32 @@
 
 /* Fullscreen CSS */
 :-webkit-full-screen.moon-video-player {
-	.full-screen-video-player;
+	.full-screen-video-player();
```

Comment:
Functionally they're the same (as far as LESS cares). The trailing `()` are useful to visually differentiate a basic mixin from a mixin-function with no arguments.